pkg/glusterfs/dockerfile/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 The Gluster CSI Authors.
```

Comment:
Again I think we can roll our own Docker files, so maybe we do not need this copyright?

Reply:
Its our own dockerfile, so the copyright. but can be removed as well.